src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -95,6 +96,19 @@ static Color GetThreadStateColor(ThreadStateSlice::ThreadState state) {
   }
 }
 
+[[nodiscard]] static std::string GetWakeupReason(ThreadStateSliceInfo::WakeupReason reason) {
+  switch (reason) {
+    case ThreadStateSliceInfo::WakeupReason::kNotApplicable:
+      return "Not Applicable";
+    case ThreadStateSliceInfo::WakeupReason::kUnblocked:
+      return "Blocked";
```

Comment:
If the thread has woken up, this should be "unblocked" in parallel with "created".

Reply:
The full tooltip says `Was Blocked by`. I think `Blocked` makes sense here. `Unblocked` would work as well, but is a bit less clear in my opinion.

Reply:
The action is that one thread unblocks another, just like the other case is that a thread creates another. I guess you want to use "blocked" as an adjective referring to the previous state instead of as the past participle of the passive form, but that's at least inconsistent in the two versions ("was created" is the passive that describes the action).